packages/realm-server/migrations/1728561353454_create-user-billing.js

```diff
@@ -0,0 +1,164 @@
+exports.up = (pgm) => {
+  pgm.createTable('users', {
+    id: {
+      type: 'serial',
+      primaryKey: true,
+    },
+    matrix_user_id: { type: 'varchar', notNull: true, unique: true },
+    stripe_customer_id: { type: 'varchar' },
+    created_at: {
+      type: 'timestamp',
+      notNull: true,
+      default: pgm.func('current_timestamp'),
+    },
+  });
+
+  pgm.createTable('plans', {
```

Comment:
I think we will want to add and populate `stripe_plan_id` once we make them there